scripts/archive_datasets.py

```diff
@@ -67,9 +67,9 @@ def make_copy(filepath):
     if not os.path.exists(target):
         logger.info("Creating archive dataset")
         try:
-            shutil.copyfile(source, target)
-        except IOError:
-            logger.exception("Could not copy file {}".format(source))
+            os.link(source, target)
+        except (IOError, OSError):
+            logger.exception("Could not hard link to file {}".format(source))
```

Comment:
How about
`logger.exception(f"Could not hard link to file { source }")`
just a personal preference for f-strings over string.format()